vyxal/elements.py

```diff
@@ -419,7 +423,8 @@ def assign_iterable(lhs, rhs, other, ctx):
         def gen():
             yield from lhs[:rhs]
             yield other
-            yield from lhs[rhs + 1 :]
+            if rhs != -1:
```

Comment:
Does this work for all negative numbers?

Reply:
Yes, it already worked with other negative numbers. Only `-1` did not work.